src/main/java/bio/terra/service/filedata/google/firestore/FireStoreFile.java

```diff
@@ -130,4 +132,68 @@ public String toString() {
         .append("size", size)
         .toString();
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    FireStoreFile that = (FireStoreFile) o;
+    return Objects.equals(fileId, that.fileId)
+        && Objects.equals(mimeType, that.mimeType)
+        && Objects.equals(description, that.description)
+        && Objects.equals(bucketResourceId, that.bucketResourceId)
+        && Objects.equals(loadTag, that.loadTag)
+        && Objects.equals(fileCreatedDate, that.fileCreatedDate)
+        && Objects.equals(gspath, that.gspath)
+        && Objects.equals(checksumCrc32c, that.checksumCrc32c)
+        && Objects.equals(checksumMd5, that.checksumMd5)
+        && Objects.equals(size, that.size);
```

Comment:
Just for my education - why are you using Objects.equals() for all of these items? Shouldn't these all be either strings or ints, so couldn't you just do a string comparison or regular "=="?

Reply:
Objects.equals() handles null values so I went with that over using .equals...although I'm not sure if any of these fields would ever be null

Reply:
Ah, makes sense! Thanks!